anndata/base.py

```diff
@@ -1561,6 +1513,123 @@ def to_df(self) -> pd.DataFrame:
             X = self._X
         return pd.DataFrame(X, index=self.obs_names, columns=self.var_names)
 
+    def _get_expression(self, use_raw=False, layer=None):
```

Comment:
AnnData is not meant for expression data only. Hence this should be `_get_matrix` or `_get_X`, I'd say. :)

Reply:
Totally agree. Though I might just get rid of this function if I go with `obj = self.raw`